.circleci/config.yml

```diff
@@ -28,6 +28,26 @@ jobs:
       - image: mcr.microsoft.com/playwright:focal
         environment:
           NODE_ENV: development
+      - image: 'bitnami/mariadb:latest'
```

Comment:
Any reason mariadb over mysql?  Fine by me - just thinking we're on mysql on prod, so...